recipes/cm-unicode/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "cm-unicode" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://kent.dl.sourceforge.net/project/{{ name }}/{{ name }}/{{ version }}/{{ name }}-{{ version }}-ttf.tar.xz
+  sha256: 2609c14450f42d0bcd40203900afcb1d693521a9b24a18c65e14b6b0585ff150
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
I actually think this maybe should be two recipes / packages (or an update to the matplotlib package so that it looks in `$PREFIX/fonts`).  This recipe that just contains fonts should be `noarch: generic`. The second recipe / package / matplotlib itself can contain the softlinks to the MPL directory

Reply:
I tired: 
```
import matplotlib.font_manager
matplotlib.font_manager.findSystemFonts(fontpaths=None, fontext='ttf')
```
And at least on my system `$PREFIX/fonts` is already included.